mne/tests/test_epochs.py

```diff
@@ -435,6 +435,31 @@ def test_reject_epochs():
     n_events = len(epochs.events)
     data = epochs.get_data()
     n_clean_epochs = len(data)
+
+    # apply rejection through constructor and drop_bad_epochs() method and
+    # check if they match
+    epochs_noreject = Epochs(raw, events1, event_id, tmin, tmax, picks=picks,
+                             baseline=(None, 0))
+    epochs_noreject.drop_bad_epochs()
+    epochs_noreject.drop_bad_epochs(reject=reject, flat=flat)
+    assert_array_equal(epochs_noreject.get_data(), epochs.get_data())
+
+    # try rejection again with same parameters
+    epochs_noreject.drop_bad_epochs()
```

Comment:
so now calling epochs_noreject.drop_bad_epochs() multiple times fails? it's an API change right?